rcp/org.testeditor.rcp4/src/org/testeditor/rcp4/dialogs/NetworkConnectionSettingDialog.java

```diff
@@ -91,10 +92,16 @@
 
 	private boolean showWelcomeMessage;
 
+	private boolean ignoreConnectionState;
+
+	private Button ignoreConnectionStateSwitch;
+
 	public NetworkConnectionSettingDialog(Shell parentShell, IEclipsePreferences prefs, IEclipseContext context) {
 		super(parentShell);
 		this.prefs = prefs;
 		this.context = context;
+		ignoreConnectionState = Boolean.valueOf(
+				System.getProperty(TE_IGNORE_CONNECTION_STATE, prefs.get(TE_IGNORE_CONNECTION_STATE, "false")));
```

Comment:
Can we extract this `System.getProperty(TE_IGNORE_CONNECTION_STATE, prefs.get(TE_IGNORE_CONNECTION_STATE, "false"))` in a separate method like `getProperty(TE_IGNORE_CONNECTION_STATE, "false")`? We could reuse it for other preferences - might come in handy later.